website/src/index.tsx

```diff
@@ -7,7 +7,7 @@ import './i18n';
 import Loader from './components/Loader';
 
 const tagManagerArgs = {
-  gtmId: 'GTM-KD8TCG4',
+  gtmId: 'GTM-WKVV7F3N',
```

Comment:
Shouldn't we use same id for both website and docs?

Reply:
Or is this a different id?

Reply:
This is a gtagmanager container ID. Its a different entity that we can use for debugging and custom tagging.